src/gridstack.ts

```diff
@@ -1317,7 +1308,7 @@ export class GridStack {
           // restore any sub-grid back
           if (n.subGrid?.el) {
             itemContent.appendChild(n.subGrid.el);
-            if (!n.subGrid.opts.styleInHead) n.subGrid._updateStyles(true); // force create
+            n.subGrid._updateStyles();

```

Comment:
again if we skipped redoing styles that were global, then most likely we don't need to do anything... need to go though all calls and only call when what we upate changes... 
ONLY when cellHeigt OR margin is changed, OR number of rows for grid total height (might need to tweak that)

Reply:
Same as above, since my changes are quite conservative, I don't feel that the performances are hurting so much in removing the line. Probably better to split _updateStyles and extract _updateContainerHeight from it?